tsdb/mapper.go

```diff
@@ -443,6 +451,8 @@ func (lm *LocalMapper) rewriteSelectStatement(stmt *influxql.SelectStatement) (*
 
 // expandWildcards returns a new SelectStatement with wildcards in the fields
 // and/or GROUP BY expanded with actual field names.
+// Tags and Measurements are included in the select if a `SELECT *` is issued
+// There is no longer an explicit `GROUP BY *` when tags are used in the `SELECT` clause
```

Comment:
Not sure about this comment. It won't make much sense to people who never knew about the implicit GROUP BY. Is it worth documenting how something used to work? What is the point?